Find a functionally equivalent code:android.media.RemoteController.setSynchronizationMode:COMMENT
Method Modifier: public      
Comment:/**
 * Set the playback position synchronization mode.
 * Must be called on a registered RemoteController.
 * @param sync {@link #POSITION_SYNCHRONIZATION_NONE} or {@link #POSITION_SYNCHRONIZATION_CHECK}
 * @return true if the synchronization mode was successfully set.
 * @throws IllegalArgumentException
 */

@@ -1,5 +1,5 @@
 {
-    if ((sync != POSITION_SYNCHRONIZATION_NONE) || (sync != POSITION_SYNCHRONIZATION_CHECK)) {
+    if ((sync != POSITION_SYNCHRONIZATION_NONE) && (sync != POSITION_SYNCHRONIZATION_CHECK)) {
         throw new IllegalArgumentException("Unknown synchronization mode " + sync);
     }
     if (!mIsRegistered) {

